hexa/plugins/connector_s3/api.py

```diff
@@ -119,6 +125,15 @@ def generate_sts_user_s3_credentials(
         5. Assume the team/pipeline/.. role
     """
 
+    if principal_credentials.endpoint_url:
+        # We are pointing to a MinIO instance, keep it simple
+        return (
```

Comment:
This would grant access to all buckets in the minio instance?

Reply:
Same as S3. Openhexa support only one blob storage provider at a given time

Reply:
My question was rather "don't we want to limit the buckets that the user can access" ? 

Especially because we are in the `generate_sts_user_s3_credentials`, which gives the impression that we are generating credentials that are specific to the user.

If it's a temporary solution to test minio, then ok by me, but let's add a TODO in the code and an issue in our backlog.